app_dart/lib/src/request_handlers/push_gold_status_to_github.dart

```diff
@@ -66,17 +66,18 @@ class PushGoldStatusToGithub extends ApiRequestHandler<Body> {
       log.debug('Last known Gold status for #${pr.number} was with sha: '
           '${lastUpdate.head}, status: ${lastUpdate.status}, description: ${lastUpdate.description}');
 
-      if (lastUpdate.status == GithubGoldStatusUpdate.statusCompleted &&
-          lastUpdate.head == pr.head.sha) {
+      if ((lastUpdate.status == GithubGoldStatusUpdate.statusCompleted &&
+              lastUpdate.head == pr.head.sha) ||
+          (pr.base.ref != 'master')) {
         log.debug('Completed status already reported for this commit.');
```

Comment:
Nit: you want to update your below doc here too?

Reply:
Whoops! Looks like I just missed this. 
I'll send a quick not follow up. 😅